frontend/README.md

```diff
@@ -26,6 +26,18 @@ You can choose your own preferred webserver like for example `nginx`, `serve` or
    npm run dev
    ```
 
+## Setting up the environment variables
+
+This application requires a couple of environment variables to run.
+Setting values for these variables can be done with a method to your own liking.
+
+| Variable               | Description                                                                         |
+|------------------------|-------------------------------------------------------------------------------------|
+| VITE_API_HOST          | Url where the api is located                                                        |
```

Comment:
URL isn't a place, it's a pointer (locator)

It's URL not Url, API not api

`URL of the API` or something like that

Reply:
https://github.com/SELab-2/UGent-3/pull/356/commits/119240ab3ef52f2202b40d3ed20b1f3ac7f60de9